media-video/ffmpeg/ffmpeg7-7.1.recipe

```diff
@@ -0,0 +1,356 @@
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libmp3lame$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libopus$secondaryArchSuffix
+	lib:libspeex$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libtheoradec$secondaryArchSuffix
+	lib:libtheoraenc$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libvorbisenc$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	REQUIRES+="
+		lib:libass$secondaryArchSuffix
+		lib:libbrotlidec$secondaryArchSuffix
+		lib:libdav1d$secondaryArchSuffix
+		lib:libfdk_aac$secondaryArchSuffix
+		lib:libfontconfig$secondaryArchSuffix
+		lib:libfreetype$secondaryArchSuffix
+		lib:libfribidi$secondaryArchSuffix
+		lib:libgme$secondaryArchSuffix
+		lib:libgmp$secondaryArchSuffix
+		lib:libopenjp2$secondaryArchSuffix
+		lib:libopenmpt$secondaryArchSuffix
+		lib:libsnappy$secondaryArchSuffix
+		lib:libsoxr$secondaryArchSuffix
+		lib:libvpx$secondaryArchSuffix
+		lib:libwebp$secondaryArchSuffix
+#		lib:libx264$secondaryArchSuffix
+#		lib:libx265$secondaryArchSuffix
+		lib:libxml2$secondaryArchSuffix
+		lib:liblzma$secondaryArchSuffix
+		"
+fi
+
+PROVIDES_avdevice="
+	ffmpeg7${secondaryArchSuffix}_avdevice = $portVersionCompat
+	lib:libavdevice$secondaryArchSuffix = $libavdeviceVersionCompat
+	"
+REQUIRES_avdevice="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix == $libavcodecVersion
+	lib:libavfilter$secondaryArchSuffix == $libavfilterVersion
+	lib:libavformat$secondaryArchSuffix == $libavformatVersion
+	lib:libavutil$secondaryArchSuffix == $libavutilVersion
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	ffmpeg7${secondaryArchSuffix}_tools = $portVersion
+	cmd:ffmpeg$secondaryArchSuffix = $portVersionCompat
+	cmd:ffplay$secondaryArchSuffix = $portVersionCompat
+	cmd:ffprobe$secondaryArchSuffix = $portVersionCompat
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix == $libavcodecVersion
+	lib:libavdevice$secondaryArchSuffix == $libavdeviceVersion
+	lib:libavfilter$secondaryArchSuffix == $libavfilterVersion
+	lib:libavformat$secondaryArchSuffix == $libavformatVersion
+	lib:libavutil$secondaryArchSuffix == $libavutilVersion
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+CONFLICTS_tools="
+	ffmpeg${secondaryArchSuffix}_tools
+	ffmpeg5${secondaryArchSuffix}_tools
+	"
+
+PROVIDES_devel="
+	ffmpeg7${secondaryArchSuffix}_devel = $portVersionCompat
+	devel:libavcodec$secondaryArchSuffix = $libavcodecVersionCompat
+	devel:libavdevice$secondaryArchSuffix = $libavdeviceVersionCompat
+	devel:libavfilter$secondaryArchSuffix = $libavfilterVersionCompat
+	devel:libavformat$secondaryArchSuffix = $libavformatVersionCompat
+	devel:libavutil$secondaryArchSuffix = $libavutilVersionCompat
+	devel:libswresample$secondaryArchSuffix = $libswresampleVersionCompat
+	devel:libswscale$secondaryArchSuffix = $libswscaleVersionCompat
+	"
```

Comment:
Maybe comment these out like in ffmpeg5, seeing ffmpeg6 is still the default for now.